### ISSUE TITLE
Compatability with the new pytorch lightning

### ISSUE THREAD

**Issue by kyoungup** [salesforce/warp-drive#88]
Hi,

I have some errors trying your notebook example:
https://lightning.ai/docs/pytorch/stable/notebooks/lightning_examples/warp-drive.html

The lightning has been updateed its path to classes and the names: pytorch_lightning to lightning.
Due to the update, I think your framework should be updated accordingly.
For example, in warp_drive.training.pytorch_lightning.py, it still includes from pytorch_lightning, not from lightning.pytorch.callbacks

**Comment by Emerald01** [comment]
We actually deprecate the maintenance of lightning integration because we found its compatibility of lower versions is poor, it often changes function signatures or even names, this makes us hard to follow considering we have multiple layers from Python to CUDA. If you want to try, please use other notebooks without lightning, or our training scripts directly.

**Comment by kyoungup** [comment]
I see
